README.adoc

```diff
@@ -189,6 +189,7 @@ $ cargo run --bin=nexus -- nexus/examples/config.toml
 ...
 listening: http://127.0.0.1:12220
 ----
+Nexus can also serve the web console. Instructions for generating the static assets and pointing Nexus to them are https://github.com/oxidecomputer/console/blob/main/docs/serve-from-nexus.md[here]. Without console assets, Nexus will still start and run normally as an API. A few link:./nexus/src/external_api/console_api.rs[console-specific routes] will 404.
```

Comment:
This makes sense for now.

It's a little weird to me that you can run it in this half-built mode that we would never use in prod.  Would it make sense to incorporate the construction of the assets into the Nexus build so that we always build a Nexus that can serve the console?

Reply:
Yes, it would make sense. I agree it's weird like this. Originally I put in stricter checks, but I realized I didn't have a good enough sense of how people were going to be running Nexus to be sure it wouldn't mess people up when they're just trying to get things ready for the demo.

One thing we could do to make things simpler is check in the built assets for Nexus in the console repo, that way Nexus would only have to shallow clone and copy the files.